apps/docs/src/app/core/component-docs/avatar-group/avatar-group-docs.component.html

```diff
@@ -0,0 +1,26 @@
+<fd-docs-section-title id="individual-type" componentName="avatar-group">
+    Individual Type
+</fd-docs-section-title>
+<description>
+    Focus on Each Person (<code>type="individual"</code>). <br>
+    The group type displays overlapping avatars. Use this type if it’s important to display the sum of people at first sight, rather than the individuals.
```

Comment:
This text (the group type...) is switched.  This text should be about the individual